offsets.scss

```diff
@@ -0,0 +1,2 @@
+$default-offset: 16px;
+$zero: 0;
```

Comment:
Just use `0` without setting it to variable. Btw, the variable name should not represent its value.